src/org/labkey/test/components/glassLibrary/components/RbtCombo.java

```diff
@@ -0,0 +1,133 @@
+package org.labkey.test.components.glassLibrary.components;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.labkey.test.selenium.EphemeralWebElement;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class RbtCombo extends WebDriverComponent<RbtCombo.ElementCache>
+{
+    final WebElement _el;
+    final WebDriver _driver;
+
+    public RbtCombo(WebElement element, WebDriver driver)
+    {
+        _el = element;
+        _driver = driver;
+    }
+
+    @Override
+    public WebElement getComponentElement()
+    {
+        return _el;
+    }
+
+    @Override
+    public WebDriver getDriver()
+    {
+        return _driver;
+    }
+
+    public RbtCombo setValue(String value)
+    {
+        if (hasValue())
+            clear();
+        elementCache().input.sendKeys( value);
+
+        elementCache().option(value).click();
+        WebDriverWrapper.waitFor(()-> !isOpened() && getValue().equals(value),
+                "the value was not set, or the combo did not close", 2000);
+        return this;
+    }
+
+    public String getValue()
+    {
+        return elementCache().input.getAttribute("value");
+    }
+
+    public boolean hasValue()
+    {
+        return !getValue().isEmpty();
+    }
+
+    // open
+    public boolean isOpened()
+    {
+        return elementCache().input.getAttribute("aria-expanded").equals("true");
+    }
```

Comment:
Be mindful of the API surface of components, especially in core test components.
Make this `private` or `protected`

Reply:
I'd considered the cost/benefit of exposing it or not, then I figured it didn't hurt to let the test have access to it/seemed silly to have the functionality but not let a test access it